product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_a_dha_cluster.mdx

```diff
@@ -153,6 +164,23 @@ The following options aren't available when creating your cluster:
 
     For more information, see [Periodic maintenance](/biganimal/release/overview/updates).
 
+1. In the **Connections** section: 
+    
+   Enable **Read-only Workloads**. This feature configures a connection string you can use to direct read-only operations to shadow nodes (non-write leaders) to lighten the workload on the write leader and improve the environment's performance. 
```

Comment:
Do we need to specify like in multi region PGD cluster, user have a choice to enable the read only connection only in one data group or all data group.